.github/workflows/sync-taro.yml

```
@@ -17,7 +17,9 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Install pnpm
-        run: corepack enable pnpm
+        run: |
+        corepack enable
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 21: could not find expected ':'

(syntax-check)